integration-tests/deployment/keystone/deploy_test.go

```diff
@@ -1,74 +1,99 @@
 package keystone_test
 
 import (
-	"encoding/json"
 	"fmt"
-	"os"
 	"testing"
 
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/stretchr/testify/assert"
 	"github.com/test-go/testify/require"
+	"go.uber.org/zap/zapcore"
+	"golang.org/x/exp/maps"
 
 	chainsel "github.com/smartcontractkit/chain-selectors"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
 
 	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
-	"github.com/smartcontractkit/chainlink/integration-tests/deployment/clo"
-	"github.com/smartcontractkit/chainlink/integration-tests/deployment/clo/models"
 	"github.com/smartcontractkit/chainlink/integration-tests/deployment/keystone"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment/memory"
 	kcr "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
 func TestDeploy(t *testing.T) {
 	lggr := logger.TestLogger(t)
 
-	wfNops := loadTestNops(t, "testdata/workflow_nodes.json")
-	cwNops := loadTestNops(t, "testdata/chain_writer_nodes.json")
-	assetNops := loadTestNops(t, "testdata/asset_nodes.json")
-	require.Len(t, wfNops, 10)
-	requireChains(t, wfNops, []models.ChainType{models.ChainTypeEvm, models.ChainTypeAptos})
-	require.Len(t, cwNops, 10)
-	requireChains(t, cwNops, []models.ChainType{models.ChainTypeEvm, models.ChainTypeEvm})
-	require.Len(t, assetNops, 16)
-	requireChains(t, assetNops, []models.ChainType{models.ChainTypeEvm})
+	// sepolia; all nodes are on the this chain
+	sepoliaChainId := uint64(11155111)
+	sepoliaArbitrumChainId := uint64(421614)
+
+	sepoliaChainSel, err := chainsel.SelectorFromChainId(sepoliaChainId)
+	require.NoError(t, err)
+	// sepoliaArbitrumChainSel, err := chainsel.SelectorFromChainId(sepoliaArbitrumChainId)
+	// require.NoError(t, err)
+	// aptosChainSel := uint64(999) // TODO:
```

Comment:
This is currently disabled because non-EVMs aren't listed in chain-selectors yet